packages/deployment/src/deployers/ERC20Deployer.js

```diff
@@ -0,0 +1,40 @@
+const BaseDeployer = require('./BaseDeployer')
+const logger = require('../helpers/logger')('ERC20Deployer')
+
+module.exports = class extends BaseDeployer {
+  constructor(config, environment, output) {
+    super(environment, output)
+    this.config = config
+  }
+
+  async call() {
+    const token = this.previousDeploy[this.config.symbol]
+    const ERC20 = await this.environment.getArtifact('ERC20Mock', '@aragon/protocol-evm')
```

Comment:
We can leave another issue for this, but it might be worth adding a mock of ANTv2 so that the test network's tokens have the same `permit()` and `transferWithAuth()` functionalities.